components/Main/Exercises/Exercises.js

```diff
@@ -20,7 +20,9 @@ export default function Exercises({
   onAddExercise,
   currentWorkout,
   addExercises,
-  onDeleteExercise,
+  exerciseAdded,
+  setExerciseAdded,
```

Comment:
```suggestion
  updateAddedExercises,
```
hier kommt die Funktion dann an und dann passiert die Magic 🪄